src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/NuGet/ProjectAssetFileWatcher.cs

```diff
@@ -78,15 +78,25 @@ protected override void Initialize()
                 return;
             }
 
-            // NOTE: Project lock file path may be null
-            var projectLockFilePath = await GetProjectLockFilePathAsync(newTree).ConfigureAwait(false);
+            try
```

Comment:
@natidea Could you please add a comment here mentioning the scenario why this was needed. Would be helpful during future code reading